tox.ini

```diff
@@ -11,12 +11,7 @@ whitelist_externals=
     make
 
 commands=
-    ; Force packaging even if setup.{py,cfg} haven't changed
-    rm -rf ./kytos.egg-info/
-    python setup.py ci
-    # The build of the documentation is running outside the tox env, so it is 
-    # necessary to clear the build after running ci.
-    make clean -C docs
```

Comment:
I was going to leave this but since part of the docs isn't completely building 100% https://github.com/kytos-ng/kytos/issues/141, we can add this again once we prioritize fixing the docs build again